ibl_to_nwb/AlyxToNWB/alyx_to_nwb_converter.py

```diff
@@ -388,15 +399,25 @@ def create_acquisition(self):
                 else:
                     if i['name'] in ['raw.lf', 'raw.ap']:
                         for j, probes in enumerate(range(self.no_probes)):
+                            if i['data'][j].shape[1] > self.one_data.data_attrs_dump['electrode_table_length'][j]:
+                                if 'channels.rawInd' in self.one_data.loaded_datasets:
+                                    channel_idx = self.one_data.loaded_datasets['channels.rawInd'][j].data
+                                else:
+                                    warnings.warn('could not find channels.rawInd')
+                                    break
+                            else:
+                                channel_idx = np.array(range(i['data'][j].shape[1]))
                             self.nwbfile.add_acquisition(
-                                TimeSeries(
+                                ElectricalSeries(
                                     name=i['name'] + '_' + self.nwb_metadata['Probes'][j]['name'],
-                                    starting_time=i['timestamps'][j][0, 1],
+                                    starting_time=np.abs(np.round(i['timestamps'][j][0, 1],2)),
```

Comment:
Why are you rounding this?

Reply:
starting times for some datasets are of the order of -1e-5, so I round to nearest 2 decimals.

Reply:
Then comment that

Reply:
You mean remove that arg? I dont want to do that as some datasets might have a larger positive starting-time

Reply:
No, I mean add "starting times for some datasets are of the order of -1e-5, so I round to nearest 2 decimals" as a comment. You have very few comments and it is difficult for me to understand what you are doing in some places